AlphaWallet/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -120,7 +120,11 @@ final class SettingsViewModel {
                 }
             case .help, .tokenStandard, .version, .wallet: return nil
             }
-        }.compactMap { [lock] isOn -> Void? in
+        }.compactMap { [lock, analytics] isOn -> Void? in
+            analytics.log(action: Analytics.Action.enableDisablePasscode, properties: [
```

Comment:
It should be logged as a user property with `setUser(property:, value:)` instead

Reply:
Probably add to a new enum-case to `UserProperties`